d2l-hm-constants-behavior.html

```diff
@@ -116,9 +116,15 @@
 			},
 			assignments: {
 				assignment: 'assignment',
+				attachment: 'attachment',
+				attachmentList: 'attachment-list',
 				instructions: 'instructions',
 				file: 'file',
-				link: 'link'
+				latest: 'latest',
+				link: 'link',
```

Comment:
I take it this is `link` like "this assignment is: view this URL" kind of thing, yeah?

Reply:
Dunno! That one's pre-existing, although I would assume that it's that (I think there's both file attachments and link attachments for assignments)

Reply:
riiiight, makes sense! Just caught my eye, all is good 😀